README.md

```diff
@@ -0,0 +1,201 @@
+        <li><a href="https://jestjs.io/">Unit Testing by Jest</a></li>
+    </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Cryptocurrency dashboard Page]**
+- **[Cryptocurrency details Page]**
+- **[Search box to look for a particular crypto item]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Setup
+
+Clone this repository to your desired folder:
+
+1 - Go to the <> Code section in the repository. <br></br>
+2 - Select the https tab and copy the url provided. <br></br>
+3 - Use the git clone command in the root of the folder where you will save the project.<br></br>
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Visual Studio Code
+- Git
+- Npm
+- Node
+- React
+- Jest
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Install
+
+Amongisde the prerequisites, you will need to install the following dependencies:
+
+- react-redux
+- redux toolkit
+- babel
+- prop-types
+- react-router and react-router-dom
+- react-icons
+- uuid
+- jest
+
+All of these dependencies can be installed with [npm](https://www.npmjs.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Usage <a name="usage"></a>
+
+To run the project, execute the following command in your terminal (inside the project folder):
+
+- npm install
+- npm start
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Run Tests <a name="run-tests"></a>
+
+To run the the tests, you have to run the following command in the console (inside the project folder):
+
+- npm install
+
+And then you will have to run the following command in the console (inside the project folder):
+
+- npm test
+
+You will see the outcome in the console
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Deployment <a name="deployment"></a>
+
+To deploy this project in a local enviroment, you will have to run:
+
+- npm start
+
+Then, you will see the project run in a local host: 3000.
+Otherwise, you can see the project deployed in Render in the live demo.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+👤 **Emmanuel Hagan**
+
+- GitHub: [@hagan199](https://github.com/hagan199)
+- LinkedIn: [@emmanuel-hagan-26219a95](https://www.linkedin.com/in/emmanuel-hagan-26219a95/)
+- Twitter: [@EmmaHagan23](https://twitter.com/EmmaHagan23)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [1] **[Adding About Page]**
+- [2] **[Adding Contact page]**
+- [3] **[Adding Desktop version]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Project Presentation -->
+
+## Project Presentation <a name="project-presentation"></a>
+
+- [Presentation]
+  [Click here for Live video presentation]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
Good job working on the project requirements. But it say in the project requirements that:

1. Video presentation

Provide feedback about the video presentation in the code review summary.

The student recorded a video presenting the features of the project.
The maximum length of the video is 5 minutes.
The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
The video contains a demo of the project, highlighting all the project features.
The video contains a highlight about a piece of the code the student finds interesting or is proud of making.


Note: It's recommended that students use [Loom](https://www.loom.com/) to record to video but they can use any tool they want as long as they share an online link of the uploaded video.

2. Live version


The student deploys the application using Render, Netlify or GitHub Pages.
The project is deployed properly, the home and detail page works as expected.
A link to the deployed project is present in the README.

So it would be nice if you added a video presentation and a deploy link based on the above requirements.